trail/trail/urls.py

```diff
@@ -18,6 +18,22 @@
 from django.conf import settings
 from django.urls import path, re_path, include
 from django.views.generic import TemplateView
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="TrailBlazer API",
+      default_version='v1',
```

Comment:
We are giving a "default" version here - but where do we register what version is a particular API of? Or am I misunderstanding what this means (didn't read the docs).